app/models/poem.rb

```diff
@@ -1,5 +1,8 @@
 class Poem < ApplicationRecord
+  POEM_STATUSES = ["Rough Draft", "Ready", "Submitted", "Accepted", "Published"]
```

Comment:
You might want to freeze this array so no one else can mutate it (that happens more than you'd think 😖 )